docs/api/disability-benefits.md

```diff
@@ -305,12 +194,20 @@ Content-Type: application/json
 * Example request:
 ```
 curl /v0/disability_claims/189625/documents \
+  -H 'Origin: https://www.vets.gov' \
+  -H 'Authorization: Token token=ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789abcde' \
+  -H 'Accept: application/json' \
+  -H 'X-Key-Inflection: camel' \
   -F tracked_item=1 \
-  -F doc_type='L307'
+  -F doc_type='L307' \
```

Comment:
We changed some of these parameters
tracked_item -> tracked_item_id
doc_type -> document_type